setup.py

```diff
@@ -63,7 +63,7 @@ def apply_patches(self, base_dir):
 
     packages=['motioneye'],
 
-    install_requires=['tornado>=3.1,<6', 'jinja2', 'pillow', 'pycurl', 'minio'],
```

Comment:
The fact that you removed `minio` from dependencies is not enough for this to work. You still import it in the python code. You should import it in a `try` .. `except` clause.